compysition/director.py

```diff
@@ -145,9 +145,8 @@ def finalize_blockdiag(self):
             if not os.path.exists(img_dir):
                 os.makedirs(img_dir)
             from blockdiag.command import BlockdiagApp
-            f = open("{0}{1}{2}.diag".format(self.blockdiag_dir, os.sep, self.name),'w')
-            f.write(self.blockdiag_out)
-            f.close()
+            with open("{0}{1}{2}.diag".format(self.blockdiag_dir, os.sep, self.name),'w') as f
```

Comment:
In languages that express type, names like f are legit. if I have for example `FileObjects f<*File>` I know f stands  for file. In Python, we don't have types, so names are a bit more important for clarity. Fine here, just get's more important as code does complex.